### DISCUSSION TITLE
CSRF token is missing Error

### DISCUSSION THREAD

**Discussion by trsree** [apache/superset#11065] [General]
While integrating Superset with our webpage , we receive the following error:

Bad Request
The CSRF session token is missing.

<img width="1661" alt="Screen Shot 2020-09-25 at 12 23 55 PM" src="https://user-images.githubusercontent.com/5104605/94291786-0a9d1f00-ff2a-11ea-8974-3bae6cd04703.png">

**Comment by emellojr** [comment]
Hi @trsree, did you solve that issue? I've the same problem, but i'm using custom authentication with jwt token, the error appears when i use the basic authentication.

Can you help me with the solution?

Tks.

**Comment by federicoarcangeli** [comment]
Hi @trsree I have the same issue as you and @emellojr
